### ISSUE TITLE
Persistence docker volume

### ISSUE THREAD

**Issue by sicenul** [dimaskiddo/go-whatsapp-rest#36]
Hello,
1. Which path should I use to make persistence data when I use docker version?
2. Which path can be use to persistence the session, so I don't have to re-login when the docker restart?

Thanks in advance

**Comment by dimaskiddo** [comment]
@sicenul You can read the Dockerfile it explain where the volumes should be mounted

**Comment by dimaskiddo** [comment]
"/usr/app/config/stores" => to store session files
"/usr/app/config/uploads" => for temporary store uploaded / sent files

**Comment by sicenul** [comment]
Thank you for your response.
I have successfully execute docker with the following command:
```
$ docker run -d -v "$PWD/userdata/stores":/usr/app/config/stores -v "$PWD/userdata/uploads":/usr/app/config/uploads -p 3000:3000 go-whatsapp-rest
```
And everything is working as expected, I can send wa message successfully.

But, if I restart the docker, I have to relogin again (rescan the qrcode), and the folder "../userdata/stores" is always empty.

Please confirm: 
1. Did I do something wrong? because folder "stores" is always empty.
2. Did the relogin process was a normal thing after a docker restart?
3. Is there any way to make it "always login"? 😄 

Thank you

**Comment by dimaskiddo** [comment]
https://github.com/dimaskiddo/go-whatsapp-rest/blob/d6a00a048d98298c894d12e2853d9e1b0c9f71ab/config/production.yaml#L5-L6

Based on the configuration it should write to that folder, please make sure your local folder is writable from container side

**Comment by sicenul** [comment]
Based on Dockerfile volumes:
https://github.com/dimaskiddo/go-whatsapp-rest/blob/d6a00a048d98298c894d12e2853d9e1b0c9f71ab/Dockerfile#L33

And production.yml:
https://github.com/dimaskiddo/go-whatsapp-rest/blob/d6a00a048d98298c894d12e2853d9e1b0c9f71ab/config/production.yaml#L5-L6

So my understanding is, I have to mount those folder on container side (based on Dockerfile) to my local folder (based on production.yml). Then my docker-compose.yml looks like this:
```
version: '2'
services:
  api:
    image: gowa-rest
    container_name: gowa
#    restart: always
    ports:
      - "3000:3000"
    volumes:
      - ./config/stores:/usr/app/config/stores
      - ./config/uploads:/usr/app/config/uploads
```
To make sure it is writable, I change with `chmod 777 -R ` to my local folder: config/stores.
But still, config/stores always empty, even though I recreate container, restart container, and rescan login.
So sorry to bother you :) , please enlighten me

**Comment by dimaskiddo** [comment]
let me check it again, so sorry

**Comment by dimaskiddo** [comment]
Really late response fix, but it's now fixed as in c8a8430
Thank you 🍻
